Sources/XcodeProj/Objects/Configuration/BuildSettings.swift

```diff
@@ -1,4 +1,69 @@
 import Foundation
 
 /// Build settings.
-public typealias BuildSettings = [String: Any]
+public typealias BuildSettings = [String: BuildSetting]
+
+public enum BuildSetting: Sendable, Equatable {
+    case string(String)
+    case array([String])
+
+    var valueForWriting: String {
```

Comment:
What do you think of this using the `CustomStringConvertible` instead of a completely custom property?

Reply:
That should work fine 👍